src/crud.rs

```diff
@@ -32,23 +32,36 @@
 ///     convert_to_receiver_signal,
 /// );
 /// ```
+
+
 #[macro_export]
 macro_rules! crud {
     (
       $crud_type:ident, $i:ident, $path:expr, $get_peers:ident, $convert_to_receiver_signal:ident
     ) => {
         ::paste::paste! {
+          use chrono::{DateTime, Datelike, NaiveDateTime, Timelike, Utc};
```

Comment:
we want to avoid doing a `use` inside a macro call, because all the code that the macro spits out sits technically inside of the crate or code that calls the macro. 
This makes working with dependencies and macros a bit confusing. 

Instead, where you want to use a type (maybe there's a better way?) use the full prefix and start it with `::` in order to refer to the dependency of the macro, as opposed to a dependency of the crate that calls the macro. 
e.g. elsewhere in the file you will see: `::hdk::prelude::holo_hash::EntryHashB64`